### ISSUE TITLE
position does not update in BMW API

### ISSUE THREAD

**Issue by gomble** [bimmerconnected/bimmer_connected#318]
### Describe the issue

Hello,

i have the problem, that the "updateTime" does not update. Therefore my homeassistant shows an old location of the car.

### Expected behavior

My homeassistant has to show the actual position of the car and not the position from 2021-08-11

### Which Home Assistant version are you using?

core-2021.9.7

### What was the last working version of Home Assistant Core?

core-2021.8

### What is your region?

Rest of world

### ConnectedDrive website

- [X] I can still successfully login to the BMW ConnectedDrive website and the car status is available there.
- [X] I have ConnectedDrive enabled for my vehicle.

### Number of cars

- [ ] I have 2 or more cars linked to the ConnectedDrive account.
- [ ] I have a Mini vehicle linked to my account.

### Output of bimmer_connected fingerprint

```txt
{
  "vehicleStatus": {
    "DCS_CCH_Activation": "NA",
    "DCS_CCH_Ongoing": false,
    "position": {
      "lat": 12.3456,
      "lon": 34.5678,
      "status": "OK"
    },
    "updateTime": "2021-08-11T07:00:29+0000",
    "vehicleCountry": "DE",
    "vin": "some_vin"
  }
}

{
  "vehicles": [
    {
      "a4a": "USB_ONLY",
      "bodytype": "F10",
      "brand": "BMW",
      "breakdownNumber": "+4989358957103",
      "carCloud": "NOT_SUPPORTED",
      "chargeNow": "NOT_SUPPORTED",
      "chargingControl": "NOT_SUPPORTED",
      "climateControl": "START_TIMER",
      "climateControlRES": "NOT_SUPPORTED",
      "climateFunction": "PARK_HEATING",
      "climateNow": "ACTIVATED",
      "climateNowRES": "NOT_SUPPORTED",
      "color": "CARBONSCHWARZ METALLIC",
      "colorCode": "416",
      "countryCode": "V6",
      "dealer": {
        "city": "some_city",
        "country": "DE",
        "name": "some_name",
        "phone": "some_phone",
        "postalCode": "some_postal_code",
        "street": "some_street"
      },
      "doorLock": "ACTIVATED",
      "doorUnlock": "ACTIVATED",
      "driveTrain": "CONV",
      "egoVehiclePath": "",
      "fuelType": "DIESEL",
      "hasAlarmSystem": true,
      "hmiVersion": "ID4",
      "hornBlow": "ACTIVATED",
      "hub": "HUB_ECE",
      "intermodalRouting": "NOT_AVAILABLE",
      "ipa": "NOT_SUPPORTED",
      "lastDestinations": "NOT_SUPPORTED",
      "lightFlash": "ACTIVATED",
      "lscType": "NOT_SUPPORTED",
      "model": "535d xDrive",
      "onlineSearchMode": "MAP",
      "onlineSearchProvider": "GOOGLE",
      "puStep": "1119",
      "rangeMap": "NOT_SUPPORTED",
      "remote360": "NOT_SUPPORTED",
      "remoteSoftwareUpgrade": "NOT_SUPPORTED",
      "sendPoi": "ACTIVATED",
      "smartSolution": "NOT_SUPPORTED",
      "statisticsAvailable": false,
      "statisticsCommunityEnabled": false,
      "steering": "LH",
      "vehicleFinder": "ACTIVATED",
      "vehicleFinderRestriction": "NONE",
      "vin": "some_vin",
      "yearOfConstruction": 2015
    }
  ]
}
```

### Anything in the logs that might be useful for us?

```txt
Executing:
bimmerconnected vehiclefinder 


DEBUG:bimmer_connected.state:{'STATUS': {'vin': '---', 'updateTime': '2021-08-11T07:00:29+0000', 'position': {'lat': ---, 'lon': ---, 'status': 'OK'}, 'vehicleCountry': 'DE', 'DCS_CCH_Activation': 'NA', 'DCS_CCH_Ongoing': False}, 'LAST_TRIP': {}, 'ALL_TRIPS': {}, 'CHARGING_PROFILE': {}, 'DESTINATIONS': {}, 'RANGEMAP': {}, 'EFFICIENCY': {}, 'NAVIGATION': {}}
DEBUG:bimmer_connected.state:received new data from connected drive
ExecutionState.EXECUTED
```

**Comment by gomble** [comment]
With pip 0.8.0.0b6 i dont geht lat and lon. It shows old values.

`bimmerconnected vehiclefinder --- --- rest_of_world WBA5D---
DEBUG:bimmer_connected.account:Getting vehicle list
DEBUG:bimmer_connected.account:Authenticating against GCDM with MyBMW flow.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): customer.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/authenticate HTTP/1.1" 200 432
DEBUG:bimmer_connected.account:got authorization challenge faNs_CNCy4K04INHvcLlWq3tANQ.*AAJTSQACMDIAAlNLABxTY1dkbnZ2Skp2YTZ5MUNOSlpNSDRkWHgrZmc9AAR0eXBlAANDVFMAAlMxAAIw---
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/authenticate HTTP/1.1" 302 0
DEBUG:bimmer_connected.account:got login code UQOBZ4cDcBgUWPGIiKoL9lYegx0
DEBUG:bimmer_connected.account:getting new oauth token
DEBUG:urllib3.connectionpool:https://customer.bmwgroup.com:443 "POST /gcdm/oauth/token HTTP/1.1" 200 1961
DEBUG:bimmer_connected.account:got new token UkXSGvGD9zeO6PruY2j_q0N--- with expiration date 2021-11-15 11:41:01.350033DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1636965662.351736&tireGuardMode=ENABLED HTTP/1.1" 200 7088
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1636965664.472233&tireGuardMode=ENABLED HTTP/1.1" 200 2
DEBUG:bimmer_connected.remote_services:Triggering remote vehicle finder
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/WBA5D71010G248716/vehicle-finder
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/WBA5D71010G248716/vehicle-finder HTTP/1.1" 200 91
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 25
DEBUG:bimmer_connected.remote_services:current state if remote service is: PENDING
DEBUG:bimmer_connected.remote_services:getting remote service status
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "POST /eadrax-vrccs/v2/presentation/remote-commands/eventStatus?eventId=37fcc9cf-008f-4658-baba-b898731932ca HTTP/1.1" 200 26
DEBUG:bimmer_connected.remote_services:current state if remote service is: EXECUTED
DEBUG:bimmer_connected.account:Getting vehicle list
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1636965703.939644&tireGuardMode=ENABLED HTTP/1.1" 200 7088
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Request to: https://cocoapi.bmwgroup.com/eadrax-vcs/v1/vehicles
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:bimmer_connected.account:Old token is still valid. Not getting a new one.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): cocoapi.bmwgroup.com:443
DEBUG:urllib3.connectionpool:https://cocoapi.bmwgroup.com:443 "GET /eadrax-vcs/v1/vehicles?apptimezone=60.0&appDateTime=1636965705.202885&tireGuardMode=ENABLED HTTP/1.1" 200 2
ExecutionState.EXECUTED`


In the MyBMW App i see the request from python and the app shows me the right position of the car:
![IMG_4978 2021-11-15 10_08_41](https://user-images.githubusercontent.com/6769445/141763041-b9a766cc-b7f9-423c-bfe5-0348e57b600f.PNG)
![IMG_4979 2021-11-15 10_08_42](https://user-images.githubusercontent.com/6769445/141763102-76de13b3-cc06-4f6e-b8de-8d9897c7a7e6.PNG)

**Comment by gomble** [comment]
Why did you close this ? This is not working, i am getting old values if i run verhiclefinder ? Issue is not solved

**Comment by gomble** [comment]
"properties": {
            "lastUpdatedAt": "2021-09-19T14:28:19Z",

i am getting old gps locations.
The reported lat lon is from 2021-09-19T14:28:19Z

**Comment by gomble** [comment]
![image](https://user-images.githubusercontent.com/6769445/142472617-5711c2fa-2e92-4e54-8c95-cda9078d06e2.jpeg)

![image](https://user-images.githubusercontent.com/6769445/142472629-666ac4fe-a5d7-4658-841e-9d2710169898.png)

The car is on another gps position but i get this.

**Comment by rikroe** [comment]
Are you using the custom component?

If not, you have to wait until our changes are merged into Home Assistant.

If yes, you should see up to date data. If still not, please provide a new fingerprint from custom component/after being included in HA.

**Comment by gomble** [comment]
I am using latest custom component with 0.8 which was released today.

**Comment by rikroe** [comment]
Does the My BMW app only show the location of your car after you have requested it?

If so, the location data does not appear in the status of the vehicle at all.
There is a special endpoint for vehicle finder that returns the vehicles location only after active search.

This needs further development and is not yet implemented.

**Comment by gomble** [comment]
After i start service .vehiclefinder on homeassistant or if i run bimmerconnected throug the shell, i see on the app the new locatoion. But if i run bimmerconnected status i see everytime the same gps location, it is not changing.

**Comment by gomble** [comment]
Here i requestet:
![image](https://user-images.githubusercontent.com/6769445/142626268-08c0539e-b8ea-45fd-a843-a334bb39f304.png)

I can see new position in the app:
![IMG_5024](https://user-images.githubusercontent.com/6769445/142626470-c0bdb887-bc95-435b-9d8b-1a87951f8cf2.PNG)

and the histoy shows me that i have requestet new position:
![IMG_5025](https://user-images.githubusercontent.com/6769445/142626608-1dd4f5e5-58ce-4ca9-ab72-6a5312c9e6bc.PNG)

But i dont get gps attributes:
![image](https://user-images.githubusercontent.com/6769445/142626666-371e3a0c-6fda-4101-9267-b85c2852fc48.png)

and if i run bimmerconnected status. It shows now null, but before it did show everytime the old gps location of the car.

My fingerprint:

`bash-5.1# cat vehicles_v2_bmw_0.txt 
[
  {
    "a4aType": "USB_ONLY",
    "bodyType": "F10",
    "brand": "BMW",
    "capabilities": {
      "canRemoteHistoryBeDeleted": false,
      "climateNow": {
        "executionMessage": "Do you want to ventilate now? Remote functions may take a few seconds.",
        "executionPopup": {
          "executionMessage": "Do you want to ventilate/heat now? Remote functions may take a few seconds.",
          "iconId": 59733,
          "popupType": "DIALOG",
          "primaryButtonText": "Start",
          "secondaryButtonText": "Cancel",
          "title": "Start Ventilation / Heating"
        },
        "executionStopPopup": {
          "executionMessage": "Stop climate control in your vehicle now? Remote functions may take a few seconds.",
          "title": "Climate control is running"
        },
        "isEnabled": true,
        "isPinAuthenticationRequired": false
      },
      "climateTimer": {
        "isEnabled": true,
        "isPinAuthenticationRequired": false,
        "isToggleEnabled": true,
        "page": {
          "description": "By setting a start time you let the vehicle know when you plan to use it.",
          "primaryButtonText": "SEND TO VEHICLE",
          "secondaryButtonText": "DEACTIVATE AND SEND TO VEHICLE",
          "subtitle": "some_road \u2022 duration \u2022 -- EUR",
          "title": "Ventilation/heating timer"
        },
        "tile": {
          "description": "Plan start time",
          "iconId": 59774,
          "title": "Ventilation/heating timer"
        }
      },
      "horn": {
        "executionMessage": "Using your horn is only allowed in certain situations in many countries. Responsibility for the use and adherence to the respective regulations lies solely with you as the user. \n\nDo you want to use the horn now? Remote functions may take a few seconds.",
        "isEnabled": true,
        "isPinAuthenticationRequired": false
      },
      "isBmwChargingSupported": false,
      "isCarSharingSupported": false,
      "isChargeNowForBusinessSupported": false,
      "isChargingHistorySupported": false,
      "isChargingHospitalityEnabled": false,
      "isChargingLoudnessEnable": false,
      "isChargingPlanSupported": false,
      "isChargingPowerLimitEnable": false,
      "isChargingSettingsEnabled": false,
      "isChargingTargetSocEnable": false,
      "isCustomerEsimSupported": false,
      "isDCSContractManagementSupported": false,
      "isDataPrivacyEnabled": false,
      "isEasyChargeSupported": false,
      "isEvGoChargingSupported": false,
      "isFindChargingEnabled": false,
      "isMiniChargingSupported": false,
      "isRemoteHistorySupported": true,
      "isRemoteServicesActivationRequired": false,
      "isRemoteServicesBookingRequired": false,
      "isScanAndChargeSupported": false,
      "lastStateCall": {
        "isNonLscFeatureEnabled": true,
        "lscState": "NOT_CAPABLE"
      },
      "lights": {
        "executionMessage": "Flash headlights now? Remote functions may take a few seconds.",
        "isEnabled": true,
        "isPinAuthenticationRequired": false
      },
      "lock": {
        "executionMessage": "Lock your vehicle now? Remote functions may take a few seconds.",
        "isEnabled": true,
        "isPinAuthenticationRequired": false
      },
      "sendPoi": {
        "executionMessage": "Send POI now? Remote functions may take a few seconds.",
        "isEnabled": true,
        "isPinAuthenticationRequired": false
      },
      "unlock": {
        "executionMessage": "Unlock your vehicle now? Remote functions may take a few seconds.",
        "isEnabled": true,
        "isPinAuthenticationRequired": true
      },
      "vehicleFinder": {
        "executionMessage": "Find your vehicle now? Remote functions may take a few seconds.",
        "isEnabled": false,
        "isPinAuthenticationRequired": false
      }
    },
    "connectedDriveServices": [],
    "driveTrain": "COMBUSTION",
    "driverGuideInfo": {
      "androidAppScheme": "com.bmwgroup.driversguide.row",
      "androidStoreUrl": "https://play.google.com/store/apps/details?id=com.bmwgroup.driversguide.row",
      "iosAppScheme": "bmwdriversguide:///open",
      "iosStoreUrl": "https://apps.apple.com/de/app/id714042749?mt=8",
      "title": "BMW\nDriver's Guide"
    },
    "exFactoryILevel": "F010-15-03-502",
    "exFactoryPUStep": "0315",
    "headUnit": "ID5",
    "hmiVersion": "ID4",
    "iStep": "F010-19-11-520",
    "isLscSupported": false,
    "isMappingPending": false,
    "isMappingUnconfirmed": false,
    "model": "535d xDrive",
    "properties": {
      "checkControlMessages": [
        {
          "description": "Check tire pressure and correct if necessary. Afterwards, perform Tire Pressure Monitor reset (TPM). See Owner's Manual for further information.",
          "id": 142,
          "name": "some_name",
          "status": "OK",
          "type": "TIRE_PRESSURE"
        }
      ],
      "climateControl": {},
      "doorsAndWindows": {
        "doors": {},
        "windows": {}
      },
      "fuelLevel": {
        "units": "LITERS",
        "value": 10
      },
      "inMotion": false,
      "isServiceRequired": false,
      "lastUpdatedAt": "2021-09-19T14:28:19Z",
      "originCountryISO": "DE",
      "serviceRequired": [
        {
          "dateTime": "2022-06-01T00:00:00.000Z",
          "status": "OK",
          "type": "VEHICLE_TUV"
        },
        {
          "dateTime": "2022-11-01T00:00:00.000Z",
          "distance": {
            "units": "KILOMETERS",
            "value": 14000
          },
          "status": "OK",
          "type": "OIL"
        },
        {
          "dateTime": "2022-11-01T00:00:00.000Z",
          "distance": {
            "units": "KILOMETERS",
            "value": 14000
          },
          "status": "OK",
          "type": "VEHICLE_CHECK"
        },
        {
          "dateTime": "2022-05-01T00:00:00.000Z",
          "status": "OK",
          "type": "BRAKE_FLUID"
        }
      ]
    },
    "puStep": "1119",
    "status": {
      "checkControlMessages": [
        {
          "criticalness": "nonCritical",
          "iconId": 60117,
          "id": "142",
          "longDescription": "Check tire pressure and correct if necessary. Afterwards, perform Tire Pressure Monitor reset (TPM). See Owner's Manual for further information.",
          "state": "Low",
          "title": "Check tire pressure. Reset TPM"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 60197,
          "state": "OK",
          "title": "Engine Oil"
        }
      ],
      "checkControlMessagesGeneralState": "1 Issue",
      "doorsAndWindows": [
        {
          "criticalness": "nonCritical",
          "iconId": 59726,
          "state": "Unknown",
          "title": "All doors"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 59701,
          "state": "Unknown",
          "title": "Left front window"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 59700,
          "state": "Unknown",
          "title": "Right front window"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 59703,
          "state": "Unknown",
          "title": "Left rear window"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 59702,
          "state": "Unknown",
          "title": "Right rear window"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 59721,
          "state": "Unknown",
          "title": "Back window"
        }
      ],
      "doorsGeneralState": "Unknown",
      "fuelIndicators": [
        {
          "chargingType": null,
          "iconOpacity": "high",
          "infoIconId": 59930,
          "infoLabel": "Fuel Level",
          "isCircleIcon": false,
          "isInaccurate": true,
          "levelIconId": 59682,
          "levelUnits": "l",
          "levelValue": "10",
          "mainBarValue": 0,
          "rangeIconId": 59681,
          "rangeUnits": "km",
          "rangeValue": "- -",
          "secondaryBarValue": 0,
          "showsBar": false
        }
      ],
      "lastUpdatedAt": "2021-09-19T14:28:19Z",
      "recallExternalUrl": null,
      "recallMessages": [],
      "requiredServices": [
        {
          "criticalness": "nonCritical",
          "iconId": 60111,
          "id": "VehicleAdmissionTest",
          "longDescription": "Next state inspection due by the specified date.",
          "subtitle": "some_road \u2022 duration \u2022 -- EUR",
          "title": "Vehicle Inspection"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 60197,
          "id": "Oil",
          "longDescription": "Next service due after the specified distance or date.",
          "subtitle": "some_road \u2022 duration \u2022 -- EUR",
          "title": "Engine oil"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 60215,
          "id": "VehicleCheck",
          "longDescription": "Next vehicle check due after the specified distance or date.",
          "subtitle": "some_road \u2022 duration \u2022 -- EUR",
          "title": "Vehicle check"
        },
        {
          "criticalness": "nonCritical",
          "iconId": 60223,
          "id": "BrakeFluid",
          "longDescription": "Next service due by the specified date.",
          "subtitle": "some_road \u2022 duration \u2022 -- EUR",
          "title": "Brake fluid"
        }
      ],
      "timestampMessage": "Updated from vehicle 9/19/2021 03:28 PM"
    },
    "telematicsUnit": "TCB1",
    "themeSpecs": {
      "vehicleStatusBackgroundColor": {
        "blue": 77,
        "green": 38,
        "red": 31
      }
    },
    "vin": "some_vin",
    "year": 2015
  }`

**Comment by jonatious** [comment]
I have the same issue. I am using the pythong CLI command directly. The lat and long are not old values ther are just 123456 and 345678 and they are the values I am getting as well

**Comment by rikroe** [comment]
Just released [0.8.3.0b1](https://github.com/bimmerconnected/bimmer_connected/releases/tag/0.8.3b1). 

It can be installed via `pip3 install --upgrade bimmer_connected==0.8.3.0b1`. When using the vehicle finder from Python (including Home Assistant), the return value from the vehicle finder service is now stored. 

I have just updated the custom component as well to try it out.

**Comment by gomble** [comment]
> Just released [0.8.3.0b1](https://github.com/bimmerconnected/bimmer_connected/releases/tag/0.8.3b1).
> 
> It can be installed via `pip3 install --upgrade bimmer_connected==0.8.3.0b1`. When using the vehicle finder from Python (including Home Assistant), the return value from the vehicle finder service is now stored.
> 
> I have just updated the custom component as well to try it out.

Okay i will check and give you feedback, thank you !

**Comment by rikroe** [comment]
Oops, there is still an issue I haven't seen. I'm on it fixing, so please wait some time!

**Comment by rikroe** [comment]
Please try with the newest custom component.

**Comment by gomble** [comment]
![image](https://user-images.githubusercontent.com/6769445/143678296-09ef742f-fa5c-4b47-9078-bfcf83909a46.png)
Did not work
updatet custom component and 
pip install bimmer-connected==0.8.3.0b4

after that restarted core 
did run the service 
bmw_connected_drive.find_vehicle

**Comment by rikroe** [comment]
Please check if there are error messages in the HA log. If using the custom component, a manual pip install shouldn't be needed.

**Comment by gomble** [comment]
> 

my bad gps in car was deactivatet, i did some tests a few days earlier. Now activated and it works again yayyyy thank you man !

**Comment by rikroe** [comment]
Thanks for confirming! I'll see that we have this integrated into the next major HA version. 

Also, could you please post fingerprints of your car with both GPS enabled and disabled?
I would like to see if it is possible to get the information from vehicle status.

**Comment by gomble** [comment]
Okay i will send you today both

**Comment by rikroe** [comment]
I think I just found the switch: In the fingerprint provided earlier, the following is set:
```json
      "vehicleFinder": {
        "executionMessage": "Find your vehicle now? Remote functions may take a few seconds.",
        "isEnabled": false,
        "isPinAuthenticationRequired": false
      }
```

All other fingerprints that were submitted to us have `"isEnabled": true`. I'll update the library and the component accordingly!

**Comment by gomble** [comment]
yes this is it 👍 many thanks to you

**Comment by rikroe** [comment]
Closing this as it is working with the latest/library custom component and has been merged into HA release 2021.12.
